core/src/main/java/io/questdb/griffin/model/QueryModel.java

```diff
@@ -498,6 +549,11 @@ public void clearOrderBy() {
         orderByDirection.clear();
     }
 
+    public void clearPivot() {
+        pivotFor = null;
```

Comment:
we need to avoid GC and "clear" but removing list elements rather than destroying the reference